### PR TITLE
[Feature] Mods Button On Main Menu Not Translatable

### LINKED ISSUE #174: [Delver] Mods Button On Main Menu Not Translatable
## Summary
The mods button on the main menu cannot be translated yet.

### PR DESCRIPTION
**Pull request by PythooonUser**
grafik

## Summary
Fixes #174. Adds `screens.MainMenuScreen.modsButton` string translation for the mods button on the main menu.

And some :lipstick:s.

### CODE REVIEW

File: Dungeoneer/src/com/interrupt/dungeoneer/screens/MainMenuScreen.java [hunk 1]
```diff
@@ -593,15 +584,24 @@ public void clicked(InputEvent event, float x, float y) {
...
+        GameApplication.SetScreen(new OverlayWrapperScreen(new ModsOverlay()));
+    }
 	
 	private void loadSavegames() {
-		String baseSaveDir = "save/";
-		FileHandle dir = Game.getFile(baseSaveDir);
+		FileHandle dir = Game.getFile(BASE_SAVE_DIR);
 		
-		Gdx.app.log("DelverLifeCycle", "Getting savegames from " + dir.path());
+		Gdx.app.log(LOG_TAG_DELVER_LIFE_CYCLE, "Getting savegames from " + dir.path());
```
Comment (joshuaskelly) [inline Dungeoneer/src/com/interrupt/dungeoneer/screens/MainMenuScreen.java:602]: I'd prefer to keep the string literal here, unless you feel strongly about it?

**Review by joshuaskelly — APPROVED** [review]
One small quesion, otherwise looks good! 👍

File: Dungeoneer/src/com/interrupt/dungeoneer/screens/MainMenuScreen.java (hunk 1 above)
Comment (PythooonUser) [reply Dungeoneer/src/com/interrupt/dungeoneer/screens/MainMenuScreen.java:602]: Oh yeah. That was past-me trying to silence SonarLint. The standard of having the string literals directly where they are needed looks definitely better.

Comment (PythooonUser) [reply Dungeoneer/src/com/interrupt/dungeoneer/screens/MainMenuScreen.java]: Fixed. Thanks!

**Review by joshuaskelly — APPROVED** [review]
Looks good! Thank you for addressing my feedback.

**Pull request merged**